web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -1310,12 +1356,17 @@ form#views-exposed-form-job-board-page,
 form#views-exposed-form-employer-directory-page {
   border-radius: 0;
   box-shadow: none;
+  padding: 10px;
 
   .card-actions, .form-actions {
     text-align: center;
 
     input.button {
-      width: 50%;
+      width: 10%;
```

Comment:
This might be better as a px width? 10% on a phone screen will be very small

Reply:
Done.